tox.ini

```diff
@@ -31,6 +31,16 @@ basepython = python2.7
 deps = https://github.com/willkg/dennis/archive/aca57f6d.zip
 commands = dennis-cmd lint --errorsonly locale/
 
+[testenv:docker]
+commands = 
+    docker-compose -f docker-compose.yml -f scripts/docker-compose.travis.yml up -d
+    docker exec -i kuma_web_1 make compilejsi18n collectstatic
+    # TODO: mysql needs some time to set up. find a way to eliminate the waiting time
+    sleep 60
```

Comment:
There are several ways to do this, but I'm not sure what is available in TravisCI. ``sleep 60`` is ok for v1, we can iterate.